docs/monthly-meeting/2022-08.md

```diff
@@ -0,0 +1,82 @@
+# Documentation Community Team Meeting (August 2022)
+
+- **Date:** 2022-08-01
+- **Time:** [21:00 UTC]<!--(https://arewemeetingyet.com/UTC/2022-08-01/21:00/Docs%20Meeting)-->
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- **Discourse thread:**
+- **Calendar for future meetings:** (send your e-mail to Mariatta for a calendar invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  XXX how to edit the hackMD doc
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+## Roll call
+
+- Mariatta / `@Mariatta`
+- Petr Viktorin / `@encukou`
+- Hugo van Kemenade / `@hugovk`
+- Ned Batchelder / `@nedbat`
+- Ezio Melotti / `@ezio-melotti`
+- CAM Gerlach / `@CAM-Gerlach`
+- Jim DeLaHunt / `@JDLH`
+- Daniele Procida
+- Pradyun Gedam
+- Erlend Aasland / `@erlend-aasland`
+
+## Discussion
+
+### How to edit HackMD doc?
+
+- Log in (with GitHub or something)
+- Then use the pencil button at the top of the page
+
+### Diataxis Workshop
+
+- 70 ppl signed up, ~35 are accepted.
```

Comment:
```suggestion
- 70 ppl signed up, ≈35 are accepted.
```

I assume you mean approximately, not proportionally here

Reply:
~ stands for "around" here.

Reply:
For some historical background, `≈` is the [approximate symbol](https://en.wikipedia.org/wiki/Approximation#Typography); `~` (tilde) was originally intended and used as a diacritic intended for overprinting over other letters for non-Latin characters, but historically was used informally for (among other purposes) an approximation (heh) of the actual approximate symbol before Unicode existed. And of course, when writing shorthand notes, its faster to type than an alt/option code.

Reply:
I never knew ~ can mean proportionally, but it's widely understood as around/about/approximately, let's keep it simple :)

https://en.wikipedia.org/wiki/Tilde#Common_use_in_English

≈ seems more of a mathematical thing.

https://en.wikipedia.org/wiki/Approximation#Typography

Reply:
I don't think I've ever seen ≈ used as a unary operator.
In formal writing this would be spelled out, in a proper mathematics text the operator would be defined. But these are shorthand notes.

Reply:
≈ is unusual and thus slightly harder to read for me personally (though I can type it just fine), and the suggestion is a “tiny nitpick” in notes, so I'll take the liberty to merge without it.
But feel free to continue the discussion if you want.